src/Imdb/Title.php

```diff
@@ -2994,16 +2994,15 @@ public function filmingDates()
     public function alternateVersions()
     {
         if (empty($this->moviealternateversions)) {
-            $page = $this->getPage('AlternateVersions');
-
-            if (false !== strpos($page, 'id="no_content"')) {
+            $xpath = $this->getXpathPage("AlternateVersions");
+            if ($xpath->evaluate("//div[contains(@id,'no_content')]")->count() ) {
                 return array();
             }
-
-            if (preg_match_all('!<div class="soda (odd|even)">\s*(.*?)\s*</div>!ims', $page, $matches)) {
-                foreach ($matches[2] as $match) {
-                    $this->moviealternateversions[] = trim(str_replace("\n", " ", $match));
-                }
+            $check = array("<ul><li>", "</li><li>");
```

Comment:
Don't now if this is the cause for PHP 7.3 crashing, but I think this one should be written as `array("<ul>", "<li>");` or `"<ul><li>";`